rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -765,9 +765,15 @@ rm -f %{_builddir}/file.list.%{name}
 %dir %{rudderlogdir}/install
 
 %if 0%{?rhel} != 3 && "%{?_os}" != "aix"
-%config(noreplace) /etc/ld.so.conf.d/rudder.conf
+%config /etc/ld.so.conf.d/rudder.conf
 %endif
 
+%config /opt/rudder/etc/uuid.hive
```

Comment:
I think this should be "noreplace" otherwise we will replace user's uuid.hive files with the empty default on each upgrade